docker/compose/docker-compose-tests.yml

```diff
@@ -15,8 +15,8 @@ services:
       - base
     restart: always
     command: bin/elasticsearch
-    # ports:
-    #   - 9200:9200
+    ports:
```

Comment:
I think we want this to remain commented out, so that you can run tests while also running an instance of mozdef in docker, which can be useful for doing dev. If you want to run ES + Rabbit exposed for testing purposes, you can run `make run-tests-resources-external`

Reply:
Addressed in [recent commit](https://github.com/mozilla/MozDef/pull/1576/commits/bf3c595806d80188101aebcdec9b61a183d51c33).